### DISCUSSION TITLE
SpeechToText - lets add it!

### DISCUSSION THREAD

**Discussion by KSemenenko** [CommunityToolkit/Maui#727] [New Feature Discussions]
we have TextToSpech in mauim so maybe it's good idea to add SpeechToText into toolkit first.

**Comment by VladislavAntonyuk** [comment]
It's a paid functionality provided by many services. You can use Azure text to speech with 5 free hours. https://azure.microsoft.com/en-us/products/cognitive-services/speech-to-text/

**Reply by KSemenenko** [reply]
not exactley,

https://developer.apple.com/documentation/speech
https://developer.android.com/reference/android/speech/SpeechRecognizer
https://learn.microsoft.com/en-us/uwp/api/windows.media.speechrecognition?view=winrt-22621

so as you can see, it's just api for each OS

**Reply by VladislavAntonyuk** [reply]
I remember I played with it on Windows, but it had a limited supported languages (English, German, French, Italian, Spanish and 2 more). And it worked offline.

I will investigate it.
Thank you for providing links.

**Reply by VladislavAntonyuk** [reply]
https://github.com/VladislavAntonyuk/MauiSamples/tree/main/MauiSpeech

We'll discuss it on next standup

**Reply by KSemenenko** [reply]
nice, so cool, do you know when will be next standup?

**Reply by VladislavAntonyuk** [reply]
It happens each first Thursday of the month, which seems like the 1st of December
